src/ProjectOrigin.WalletSystem.Server/Repositories/CertificateRepository.cs

```diff
@@ -331,4 +331,95 @@ public Task<Claim> GetClaim(Guid claimId)
     {
         return _connection.QuerySingleAsync<Claim>("SELECT * FROM claims WHERE id = @claimId", new { claimId });
     }
+
+    public async Task<IEnumerable<ClaimViewModel>> GetClaims(string owner, ClaimFilter claimFilter)
+    {
+        string sql = @"
+        CREATE TEMPORARY TABLE claims_work_table ON COMMIT DROP AS (
```

Comment:
Why the temp table?

Reply:
It executes two queries, one for the claims, and one for all the attributes that relates to the certificates, it is no feasible to use to join all the attribute onto aswell, so in this manner, all attributes relating to the certificates can be easily found, and the table is droppen when the transaction ends.